easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'TetGen'
+version = '1.5.0'
+
+homepage = 'http://wias-berlin.de/software/index.jsp?id=TetGen'
+description = """ A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator """
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```

Comment:
Any particular reason you're not building this with Intel compilers?

Reply:
It allows to use it with both the foss and intel chains (I intend to use it with foss).